mise.toml

```diff
@@ -29,9 +29,9 @@ just = "1.37.0"
 # Foundry is a special case because it supplies multiple binaries at the same
 # GitHub release, so we need to use the aliasing trick to get mise to not error
 # The git ref here should be on the `stable` branch.
-forge = "nightly-5d16800a64e5357fbb2493e4cae061756d145981"
-cast = "nightly-5d16800a64e5357fbb2493e4cae061756d145981"
-anvil = "nightly-5d16800a64e5357fbb2493e4cae061756d145981"
+forge = "stable"
```

Comment:
We should ask in the foundry support channel if they plan to release `vX.Y.Z` releases we can pin to

Reply:
They created the v1.0.0 tag, updating